source/deployment/webrtc.rst

```diff
@@ -7,17 +7,65 @@ The feature is a working prototype for community development and not recommended
 
 .. note:: This feature will be replaced by a plug-in architecture allowing multiple video and audio calling providers to connect into Mattermost, and features described in this section will be re-written as a plug-in.
 
-Configuring video and audio calls
-------------------------------------------
+Configuring video and audio calls with WebRTC
+----------------------------------------------
 
-This option can be enabled by the System Administrator in the System Console under **Integrations > WebRTC (Beta)** - `see configuration settings documentation to learn more <https://docs.mattermost.com/administration/config-settings.html#webrtc-beta>`_.
+This guide is aimed to set up Mattermost WebRTC with a Docker image, `available here <https://hub.docker.com/r/mattermost/webrtc/>`_  and a Janus server to act as the WebRTC gateway. 
 
-To set up the WebRTC server, you may either
+If you want to use a full `Janus Gateway <https://janus.conf.meetecho.com/>`_, please visit their `GitHub repo <https://github.com/meetecho/janus-gateway>`_ for detailed instructions. You may also optionally set up `Coturn <https://github.com/coturn/coturn/wiki>`_ for STUN and TURN servers for your Mattermost installation.
 
- - use a `Mattermost Docker container created for testing WebRTC <https://hub.docker.com/r/mattermost/webrtc/>`_
- - set up a `Janus server <https://github.com/meetecho/janus-gateway>`_ to act as the WebRTC gateway and `Coturn <https://github.com/coturn/coturn/wiki>`_ for STUN and TURN servers for your Mattermost installation.
+Pre-requisites
+~~~~~~~~~~~~~~~
 
-Starting a video call
+- Install Docker using the `Ubuntu online guide <https://docs.docker.com/installation/ubuntulinux/`_. Docker installation, configuration and management are out of scope for this guide.
+- Install a Mattermost server using the `official install guides <https://docs.mattermost.com/guides/administrator.html#installing-mattermost>`_. Mattermost server installation, configuration and management are out of scope for this guide.
+- Install Janus server version 0.2.2.
+- Add an SSL certificate for host **dockerhost**, valid until January 2, 2018.
+- Ability to connect using SSL or plain WebSocket and HTTP // XXX Is this for WebRTC connection?
+
+STUN and TURN servers are not required for this setup.
```

Comment:
Incorrect you need at least one stun server that is why I recommended the google public one